ocaml/xenopsd/c_stubs/xenctrlext_stubs.c

```diff
@@ -146,8 +146,8 @@ CAMLprim value stub_xenctrlext_get_runstate_info(value xch_val, value domid)
 
 CAMLprim value stub_xenctrlext_get_boot_cpufeatures(value xch_val)
 {
-	CAMLparam1(xch_val);
 #if defined(XENCTRL_HAS_GET_CPUFEATURES)
+	CAMLparam1(xch_val);
 	CAMLlocal1(v);
 	uint32_t a, b, c, d, e, f, g, h;
```

Comment:
It looks like `XENCTRL_HAS_GET_CPUFEATURES` and `xc_get_boot_cpufeatures` was always local to the XenServer patchqueue, and deleted by https://github.com/xenserver/xen.pg/commit/b39aebaf663395714ffcff19e9707500a1580d3f#diff-6c3fe5afba8527c7d13d37280dbc422a422f357c226fc8d1759031a10a32aa20L37 in 2015.

It's probably worth cleaning up the dead code here too